### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by Chunbear** [Zetaphor/PromptHub#2]
webui has updated gradio 3.23, the UI interface has changed and cannot be used normally
![image](https://user-images.githubusercontent.com/34836688/227728134-3f456768-4a54-4cc0-b6dd-6a3e9af372e8.png)

**Comment by Zetaphor** [comment]
Yeah this update today broke a LOT of extensions, I'm working on a fix now

**Comment by mccabedd** [comment]
Hi, Am wondering if there is any update on the fix for this?
